src/components/About/About.module.css

```diff
@@ -1,3 +1,139 @@
 .title {
   text-align: center;
 }
+
+@keyframes ldio {
+    0% { opacity: 1 }
+    100% { opacity: 0 }
+  }
+  .ldio div {
+    left: 94px;
+    top: 48px;
+    position: absolute;
+    animation: ldio linear 1s infinite;
+    background: #fe718d;
+    width: 12px;
+    height: 24px;
+    border-radius: 6px / 12px;
+    transform-origin: 6px 52px;
+  }.ldio div:nth-child(1) {
+    transform: rotate(0deg);
+    animation-delay: -0.9166666666666666s;
+    background: #fe718d;
+  }.ldio div:nth-child(2) {
+    transform: rotate(30deg);
+    animation-delay: -0.8333333333333334s;
+    background: #fe718d;
+  }.ldio div:nth-child(3) {
+    transform: rotate(60deg);
+    animation-delay: -0.75s;
+    background: #fe718d;
+  }.ldio div:nth-child(4) {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5763907)